README.md

```diff
@@ -33,6 +33,8 @@ We run our own private registry on a server with limited storage and it was only
       username for docker login
 -password string
       password for docker login
+-promote
+      promote user to enter docker login password; If specified will ignore `-password`
```

Comment:
I'd leave this out and see if any value is given after `-password` or not. If not, prompt user.